bin/near

```diff
@@ -84,20 +65,44 @@ const newProject = {
   command: 'new_project',
   desc: 'create a new blank project',
   builder: (yargs) => yargs
-    .option('project_dir', {
+    .option('projectDir', {
       desc: 'project directory',
       type: 'string',
       default: '.'
     }),
   handler: (argv) => exitOnError(main.newProject(argv))
 };
 
+const config = require(process.cwd() + '/src/config')(process.env.NODE_ENV || 'development');
 yargs // eslint-disable-line
+  .option('nodeUrl', {
+    desc: 'NEAR node URL',
+    type: 'string',
+    default: 'http://localhost:3030'
+  })
+  .option('helperUrl', {
+    desc: 'NEAR contract helper URL',
+    type: 'string',
+    default: 'http://localhost:3030'
```

Comment:
should this be :3000?

Reply:
should actually just be `null` I think (node should be used directly locally)